misc/landing-page/src/components/Steps.astro

```diff
@@ -3,33 +3,37 @@ import Step from "./Step.astro";
 import Section from "./Section.astro";
 ---
 
-<Section id="features" background="light" class="gap-36">
-    <Step n="01">
-        <Fragment slot="title">Compile your<br /> Elixir to Wasm</Fragment>
-        <Fragment slot="description">
-            One click compilation. Infinite possibilities. Turn your Elixir into
-            WebAssembly and unlock the browser's full potential.
-        </Fragment>
-    </Step>
-    <Step n="02">
-        <Fragment slot="title">With no changes<br /> in the code</Fragment>
-        <Fragment slot="description">
-            Frontend meets functional. Your Elixir now speaks fluent browser
-            with APIs that make web development feel like home.
-        </Fragment>
-    </Step>
-    <Step n="03">
-        <Fragment slot="title">Run it in the<br /> browser</Fragment>
-        <Fragment slot="description">
-            Your code. Unchanged. Unstoppable. Popcorn runs your existing Elixir
-            through AtomVM magic – no refactoring required.
-        </Fragment>
-    </Step>
-    <Step n="04">
-        <Fragment slot="title">Be sure it is<br /> stable</Fragment>
-        <Fragment slot="description">
-            Enterprise stability. Startup speed. Our reinforced Elixir-JS bridge
-            delivers rock-solid performance when it matters most.
-        </Fragment>
-    </Step>
+<Section id="features" background="light" class="md:gap-36 gap-20">
+  <Step n="01">
+    <Fragment slot="title"
+      >Compile your<br class="break" /> Elixir to Wasm</Fragment
+    >
+    <Fragment slot="description">
+      One click compilation. Infinite possibilities. Turn your Elixir into
+      WebAssembly and unlock the browser's full potential.
+    </Fragment>
+  </Step>
+  <Step n="02">
+    <Fragment slot="title"
+      >With no changes<br class="break" /> in the code</Fragment
```

Comment:
This was unexpected, I didn't see custom prop until end of PR. We can just remove `br` if it's not needed on smaller screens.